packages/next-swc/crates/napi/src/next_api/project.rs

```diff
@@ -791,7 +791,9 @@ pub fn project_update_info_subscribe(
 #[derive(Debug)]
 #[napi(object)]
 pub struct StackFrame {
+    // 1-indexed, unlike source map tokens
```

Comment:
That's very unusual... Sure that it is that way?

Reply:
Yeah. Error stacks in JS are 1-based, and the stack parsing library we use doesn’t change that.